README.adoc

```diff
@@ -91,6 +92,66 @@ When a rule matches, the Authorizer will return that rule’s evaluation as a de
 When no rules match, then by default it is denied.
 So, if ACL is not defined for a resource type then it automatically gets denied.
 
+== ManagedKafkaCreateTopicPolicy
+
+The objective of this policy is to:
+
+* validate the topic replication factor (default allows 3 only)
+* validate number of partitions in the cluster
+* validate the create topic configs
+
+Here are the configuration description:
+
+[cols="2,1,5"]
+|===
+|property |default value |description
+
+|kas.policy.create-topic.max.partitions |-1 |Specifies the upper limit of partitions that should be allowed in the cluster. If this property is not specified, the default of `-1` implies no limit.
+|kas.policy.create-topic.partition-counter.timeout-seconds |10 |Specifies the duration in seconds to use as a timeout when listing and describing topics. Defaults to 10.
+|kas.policy.create-topic.partition-counter.private-topic-prefix |"" |Specifies the prefix used to match against topic name to detect private/internal topics. Defaults to blank - no prefix.
+|kas.policy.create-topic.partition-counter.schedule-interval-seconds |15 |Specifies the interval (in seconds) between partition counts. Defaults to 15.
+|kas.policy.create-topic.partition-limit-enforced |false |`true` disables the enforcement of the partition limit. Defaults to `false` (limit enforced).
+|kas.policy.topic-config.enforced |"flush.ms:9223372036854775807,index.interval.bytes:4096,..."[1] |Specifies a mapping between a topic policy key and a value. The property is expressed as a comma separated list of `key:value` pairs where the key and value are separated by  a `:`. Hence the it takes the form `keyA:value1,keyB:value2`.
+|kas.policy.topic-config.mutable |"message.timestamp.difference.max.ms,message.timestamp.type,..."[2] |Specifies which properties of topic are allowed to be configured. It's a comma separated list. The list will be merged with `kas.policy.topic-config.enforced` and `kas.policy.topic-config.range config`.
+|kas.policy.topic-config.range |"max.message.bytes::1048588,segment.bytes:52428800:,segment.ms:600000:" |Specifies a mapping from config key to an inclusive/closed range off values. Entries are formatted as a triplet using `:` as a separator, `key:min:max` where either min or max are optional implying a single bound as appropriate. Multiple triplets are comma separated. For example  `configA:minA:maxA,configB:minB:maxB,...`.
+|===
+
+[1]"flush.ms:9223372036854775807,index.interval.bytes:4096,compression.type:producer,flush.messages:9223372036854775807,segment.jitter.ms:0,min.cleanable.dirty.ratio:0.5,file.delete.delay.ms:60000,segment.index.bytes:10485760,preallocate:false,unclean.leader.election.enable:false"
+[2]"message.timestamp.difference.max.ms,message.timestamp.type,retention.bytes,min.compaction.lag.ms,cleanup.policy,max.compaction.lag.ms,delete.retention.ms,message.downconversion.enable"
```

Comment:
Put these 2 value out of the table, to avoid occupying too many table width size.

Reply:
Re-ordering the columns so the description is before the default might make that less of an issue.